packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -184,22 +184,6 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 					helpers.exportAs(ids, 'png', getExportName(editor, defaultDocumentName))
 				},
 			},
-			{
```

Comment:
so, i agree with these being only useful for debugging and not practically useful. (and the confusion with the .tldr json format)

howeeeever...before you remove them, would it make sense to have the snapshot json format be available? it feels like vendor-lock-in to not have a proper non-lossy export option, ya know?

Reply:
the snapshot json format is available through `editor.getSnapshot` / `editor.loadSnapshot` or the `snapshot` prop on the `<Tldraw />` component

Reply:
there's also the getcontent/putcontent APIs which are used by the clipboard

Reply:
I know that, programmatically, those are available on the APIs. The problem is now that they're hidden from UI by default, which effectively neuters their efficacy. 
To be clear, I'm saying we need to keep some UI around this out-of-the-box.